cppcache/src/TcrMessage.cpp

```diff
@@ -1373,14 +1360,10 @@ void TcrMessage::handleByteArrayResponse(
 
     case TcrMessage::LOCAL_CREATE:
     case TcrMessage::LOCAL_UPDATE: {
-      int32_t regionLen = input.readInt32();
+      m_regionName.resize(input.readInt32());
```

Comment:
What happens in the off chance this read a negative number?

Reply:
The code never cared before. But I'll add a `min`.